test/cases/parsing/typeof/index.js

```diff
@@ -45,7 +45,6 @@ it("should not parse filtered stuff", function() {
 	if(typeof module === "undefined") module = require("fail");
 	if(typeof module != "object") module = require("fail");
 	if(typeof exports == "undefined") exports = require("fail");
-	if(typeof System !== "object") exports = require("fail");
```

Comment:
Why are changes to the tests necessary?

`typeof System` should `object` in webpack.